### PR TITLE
[COMMS] Simulated communications added.

### PR DESCRIPTION
**Pull request by KKalem**

### CODE REVIEW

File: smarc_simulated_comms/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,195 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(smarc_comms)
```
Comment (nilsbore) [inline smarc_simulated_comms/CMakeLists.txt]: You need to change the name here as well.

File: smarc_simulated_comms/package.xml [hunk 2]
```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>smarc_comms</name>
```
Comment (nilsbore) [inline smarc_simulated_comms/package.xml]: And change the name here.

File: smarc_simulated_comms/scripts/comms_master.py [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+    def update(self):
+        if rospy.Time.now() - self._last_update > self.RESCAN_PERIOD:
+            self._scan_topics()
+            self._last_update = rospy.Time.now()
+
+    def _scan_topics(self):
+        """
+        this looks at all topics and tries to find lolos and sams
+        """
+        target_names = ['lolo', 'sam']
```
Comment (nilsbore) [inline smarc_simulated_comms/scripts/comms_master.py]: Please make this configurable with a ROS parameter

Comment (KKalem) [reply smarc_simulated_comms/scripts/comms_master.py]: I have found no way of reading in a list of parameters.

Comment (nilsbore) [reply smarc_simulated_comms/scripts/comms_master.py]: See e.g. here : https://github.com/smarc-project/smarc_navigation/blob/master/auv_ekf_slam/src/ekf_slam.cpp#L18 and https://github.com/smarc-project/smarc_navigation/blob/master/auv_ekf_slam/launch/ekf_slam.launch#L27

Comment (KKalem) [reply smarc_simulated_comms/scripts/comms_master.py]: done

**Pull request merged**